test-utils/generators/Generators.scala

```diff
@@ -140,8 +144,9 @@ trait Generators extends ModelGenerators with RegexConstants {
     RegexpGen
       .from(regex)
       .suchThat(
-        value => value.trim.nonEmpty && value.length <= length
+        value => value.trim.nonEmpty
       )
+      .map(_.take(length))
```

Comment:
This fixes the flakiness in `ContactNameFormProviderSpec` where generated values were getting discarded.
```
[info] - bind valid data *** FAILED ***
[info]   Gave up after 8 successful property evaluations. 51 evaluations were discarded.
```